blocks/bricks/conv.py

```diff
@@ -177,6 +180,144 @@ def num_output_channels(self):
         return self.num_filters
 
 
+class ConvolutionalTranspose(Initializable):
+    """Performs the transpose of a 2D convolution.
+
+    Parameters
+    ----------
+    filter_size : tuple
+        The height and width of the filter (also called *kernels*).
+    num_filters : int
+        Number of filters per channel.
+    num_channels : int
+        Number of input channels in the image. For the first layer this is
+        normally 1 for grayscale images and 3 for color (RGB) images. For
+        subsequent layers this is equal to the number of filters output by
+        the previous convolutional layer. The filters are pooled over the
+        channels.
+    original_image_size : tuple
+        The height and width of the output (image or feature map).
+    batch_size : int, optional
+        Number of examples per batch. If given, this will be passed to
+        Theano convolution operator, possibly resulting in faster
+        execution.
+    image_size : tuple, optional
+        The height and width of the input (image or feature map). If given,
```

Comment:
Isn't it passed for Convolutional?

On Fri, Jan 22, 2016 at 5:40 PM, vdumoulin <notifications@github.com> wrote:

> In blocks/bricks/conv.py
> <https://github.com/mila-udem/blocks/pull/946#discussion_r50600970>:
>
> > +    num_filters : int
> > +        Number of filters per channel.
> > +    num_channels : int
> > +        Number of input channels in the image. For the first layer this is
> > +        normally 1 for grayscale images and 3 for color (RGB) images. For
> > +        subsequent layers this is equal to the number of filters output by
> > +        the previous convolutional layer. The filters are pooled over the
> > +        channels.
> > +    original_image_size : tuple
> > +        The height and width of the output (image or feature map).
> > +    batch_size : int, optional
> > +        Number of examples per batch. If given, this will be passed to
> > +        Theano convolution operator, possibly resulting in faster
> > +        execution.
> > +    image_size : tuple, optional
> > +        The height and width of the input (image or feature map). If given,
>
> I'll update it. Would you also like me to update the Convolutional
> description as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/blocks/pull/946/files#r50600970>.
>